api/src/utils/media/csv/validation/csv-row-validator.ts

```diff
@@ -256,12 +255,12 @@ export const getNumericFieldsValidator = (config?: ColumnNumericValidatorConfig)
         return csvWorksheet;
       }
 
+      const rowValueForColumn = Number(row[columnIndex]);
```

Comment:
Any reason for moving this operation before the branch on L260?

Reply:
I think it was shifted while I was testing things. Moved back